developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx

```diff
@@ -17,72 +17,86 @@ import ReactPlayer from 'react-player/lazy'
 
 Now that you've set up your own Weaviate instance and installed a client, let's get hands-on with Weaviate.
 
-### <i class="fa-solid fa-code"></i>&nbsp;&nbsp;Client instantiation
+### <i class="fa-solid fa-arrow-right-from-bracket"></i>&nbsp;&nbsp; Client instantiation
 
```

Comment:
Hey, I think I prefer the code symbol to distinguish between the theory and practical symbols. Can we leave it for now?

Reply:
Updated. 

I'll update the other PR(s) after you've had a look at them.